wcfsetup/install/files/lib/acp/page/ApplicationManagementPage.class.php

```diff
@@ -1,62 +0,0 @@
-<?php
```

Comment:
You need to explicitly delete this file. Otherwise this is going to collide for upgrades.

see https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/acp/update_com.woltlab.wcf_5.4_removeFiles.php